tests/lib/snaps/snapctl-disable-services-install-hook/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: snapctl-disable-services-install-hook
+version: 1.0
+apps:
+    simple:
+        daemon: simple
+        command: "sleep 60"
+    forking:
+        daemon: forking
+        command: sh -c "sleep 60 &"
```

Comment:
Does job control work without a pty?

Reply:
I'm not sure, I originally was just testing with `yes`, but it occurred to me that if the test fails for some reason that `yes` process would just eat up CPU until it was killed which I'm not sure when that would happen. There may be a better service to try here that doesn't use up CPU but stays alive for a while